doc/tutorials/mlip/mlip.ipynb

```diff
@@ -0,0 +1,447 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #1.    for atom in etoh:
</pre><p>It isn't clear why the atoms should be in the model object. Have you introduced the type mapping? What it is, why you need it?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/espressomd/espresso/pull/4982/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='14'/>